svm40/svm40.h

```diff
@@ -124,6 +124,32 @@ int16_t svm40_read_measured_values_as_integers(int16_t* voc_index,
                                                int16_t* relative_humidity,
                                                int16_t* temperature);
 
+/**
+ * Returns the new measurement results as integers with raw values added.
```

Comment:
```suggestion
 * Returns the new measurement results as integers along with the raw gas signal and uncompensated RH/T values.
```
would you mind double checking if the datasheet calls it gas signal or voc ticks?

Reply:
It's voc_ticks so I changed it to " ... as integers along with the raw voc ticks and uncompensated RH/T values."